src/stylesheets/custom.scss

```diff
@@ -222,6 +222,10 @@
   top: -1px;
 }
 
+.sticky.top-4 {
+  top: 4rem;
+}
+
```

Comment:
`.top-4` works mainly as a utility class, we can probably leave it untethered to `.sticky`.  To behave more in line with other utility classes (both internal and USWDS) , something like this: 

```
.top-8 {
  top: 4rem !important.
}
```

USWDS writes their utilities in half increments, so 1 = 05, 2 =1, etc.  This isn't as much of an issue be just provides a known standard

Reply:
ah, good point. updating code to match feedback